ecommerce/android/Getflix/app/src/main/res/layout/fragment_home_page.xml

```diff
@@ -1,25 +1,465 @@
+                                android:text="@string/category_woman"
+                                android:textColor="#EBD494"
+                                android:textSize="16dp"
+                                app:layout_constraintBottom_toBottomOf="parent"
+                                app:layout_constraintEnd_toEndOf="parent"
+                                app:layout_constraintStart_toStartOf="parent"
+                                app:layout_constraintTop_toBottomOf="@+id/icon_woman" />
+                        </androidx.constraintlayout.widget.ConstraintLayout>
+                    </androidx.cardview.widget.CardView>
+
+                    <androidx.cardview.widget.CardView
+                        android:layout_width="0dp"
+                        android:layout_height="match_parent"
+                        android:layout_margin="10dp"
+                        android:layout_weight="1"
+                        android:shape="ring"
+                        app:cardCornerRadius="75dp"
+                        app:cardElevation="8dp">
+
+                        <androidx.constraintlayout.widget.ConstraintLayout
+                            android:layout_width="match_parent"
+                            android:layout_height="match_parent"
+                            android:background="#000000">
+
+                            <ImageView
+                                android:id="@+id/icon_man"
+                                android:layout_width="80dp"
+                                android:layout_height="80dp"
+                                android:layout_marginBottom="16dp"
+                                app:layout_constraintBottom_toBottomOf="parent"
+                                app:layout_constraintEnd_toEndOf="parent"
+                                app:layout_constraintStart_toStartOf="parent"
+                                app:layout_constraintTop_toTopOf="parent"
+                                app:srcCompat="@drawable/ic_man"
+                                tools:ignore="VectorDrawableCompat" />
+
+                            <TextView
+                                android:id="@+id/text_man"
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:text="@string/category_man"
+                                android:textColor="#EBD494"
+                                android:textSize="16sp"
+                                app:layout_constraintBottom_toBottomOf="parent"
+                                app:layout_constraintEnd_toEndOf="parent"
+                                app:layout_constraintStart_toStartOf="parent"
+                                app:layout_constraintTop_toBottomOf="@+id/icon_man" />
+                        </androidx.constraintlayout.widget.ConstraintLayout>
+                    </androidx.cardview.widget.CardView>
```

Comment:
For each card view for categories, we can use a custom view instead of repeating the code, but I think it's not an urgent task for Milestone 1. I'm saying it just for refactoring, which makes our layout files shorter.

Reply:
As far as the quality of the code concerned, I have thought using a custom view is an elegant solution. So I feel you but I have prioritized the completeness of project for Milestone 1 since we have limited time compared to countless tasks. That's why, I have written this code in a way that I could have completed it asap.